### ISSUE TITLE
Cant drag chat back right

### ISSUE THREAD

**Issue by TarAlacrin** [night/BetterTTV#1888]
So right now, if I have the chat on the default right side, and I grab it and drag it to the left it switches sides and the whole site gets flipped. Which is cool. However, I can't then drag it back to the right. 

Also, incidentally it doesn't change by Bttv settings when I drag the chat to the left. so the only way to bring the chat back to the right is to go into the settings, enable left side chat, then disable it. 

Its pretty annoying when I accidentally drag and the chat switches sides and I have to into a menu and double toggle a switch. But in the grand scheme of things its not a big deal.

**Comment by Aaron128l** [comment]
Make sure that your dragging the lower part of the chat, under the input box. 

The not changing the setting is a bug.

**Comment by TarAlacrin** [comment]
Yeah I made sure that I was dragging from the bottom. No dice.

On Fri, Dec 16, 2016, 6:05 PM Aaron L. <notifications@github.com> wrote:

Make sure that your dragging the lower part of the chat, under the input
box.

The not changing the setting is a bug.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/night/BetterTTV/issues/1888#issuecomment-267717591>,
or mute
the thread

**Comment by EhsanKia** [comment]
You actually accidentally dragged the chat to the left side? I tried to make the feature in such a way that it doesn't happens accidentally. And yes, it's meant to be temporary (refresh resets it back). If you want it permanent, you have to swap in options.

Not being able to swap back is also a big issue. Are you dragging it all the way to the right as far as you can go before letting go? Can you give more information? The size of your screen/window, high dpi device? operating system, browser, etc.

**Comment by TarAlacrin** [comment]
Yeah sure. Windows 10, chrome. When dragging to the left from the right all
I had to do was drag it like a couple pixels left and it would snap. When
dragging from the left to the right i dragged it all the way to the other
side and it wouldn't snap back.

I think I figured out what's causing the bug though: I have a 3 monitor
setup with twitch on the left screen. (All 3 screens are 1920x1080)
unfortunately I've already left work and can't verify if it functions
differently on different monitors until Monday.
